CondTools/Geometry/test/writehelpers/splitExtended2021Database.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/sh
 
-conddb --db myfile.db XMLFILE_Geometry_TagXX_Extended2021_mc XMLFILE_Geometry_TagXX_Extended2021_mc --destdb GeometryFileExtended2021.db
```

Comment:
This was a recent contribution:
https://github.com/cms-sw/cmssw/commit/3a57f73f00a710a344cb31c0f1bc01c09367738c

Reply:
That version of the script simply would not run because the wrong arguments were being passed to `conddb`. I followed the example of the other `split` scripts in the same directory to make this run.
I mentioned this problem to the developer who made the change, and he said he was aiming to fix it, but I guess I got to it first.